web/modules/custom/vote_utility/translations/nv.po

```diff
@@ -13,214 +8,285 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\\n"
 "Plural-Forms: nplurals=2; plural=(n > 1);\\n"
 
-#language selector
+# config.languages.languageName
 msgid "Navajo"
 msgstr "Diné"
 
+# config.languages.params.select_language
 msgid "Select language"
 msgstr "Saad choid77['88[7g77"
 
-#main
+# config.languages.params.Owner
 msgid "@sitename in language"
-msgstr "@sitename Diné"
+msgstr "@sitename in Diné"
```

Comment:
Remove `in`. Compare with the live site.

Reply:
removed 'in' to match live site